projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/genetics/attributes.json

```
@@ -0,0 +1,49 @@
+{
+    "name": "蜜蜂属性",
+    "icon": "productivebees:gene",
+    "category": "productivebees:genetics",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "通过将分离出来的基因放在蜜蜂小食上并将其喂给蜜蜂，可以将蜜蜂的属性从一只转移到另一只。$(br)请参阅有关$(l:genetics/collecting#crystalline)收集基因$(/l)的页面。"
+        },
+        {
+            "type": "patchouli:text",
+            "title": "生产力",
```

```suggestion
            "title": "产量",
```
title和zh_cn统一称呼，本文件通查